src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,94 @@
+@import "./utils/variables";
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  position: relative;
+  height: $stopwatch-2r;
+  width: $stopwatch-2r;
+
+  box-sizing: border-box;
+  border-radius: 50%;
+  border: $stopwatch-r * 0.01 dotted $page-color;
+  background: $page-bg-color;
+
+  &::after {
+    position: absolute;
+    content: "";
+    height: $center-2r;
+    width: $center-2r;
+
+    border-radius: 50%;
+    background: $hand-color;
+  }
+
+  &__lines {
+    position: absolute;
+    height: $line-size-a;
+    width: $line-size-b;
+
+    border-radius: 50%;
+    transform: rotateZ(90deg);
+    background: $page-color;
+
+    $pos: (
+      "top": 1,
+      "left": 2,
+      "bottom": 3,
+      "right": 4
+    );
+
+    @each $side, $i in $pos {
+      &:nth-child(#{$i}) {
```

Comment:
You can use index in each loops to get the index position. You wouldn't need to store indexes this way.
```scss
@each $item in $list{
    $i: index($item, $list);

    ...loop code here...
}

Reply:
@UmizDemud Thanks I actually didn't know that